test/test-runner.mjs

```
@@ -0,0 +1,166 @@
+
+  console.log('\nAll tests passed!');
+})();
+
+// Run tests
+(async () => {
+  console.log('Starting test suite...');
+  
+  await test('GET /ping should return 200 with timestamp and cf info', async () => {
+    const { req, ctx } = createRequest('GET', '/ping');
+    req.cf = { colo: 'DFW', country: 'US' };
+    const res = await worker.fetch(req, mockEnv, ctx);
```

ESLint

[error] 'worker' is not defined.

(no-undef)